blocks/core/ff_module/ff_module-task-event/src/TaskEventContainer.rt

```diff
@@ -0,0 +1,6 @@
+<div
+    class="ff_module-task-event ff_module-task-event--{this.props.event.type}"
+    >
+    <time class="ff_module-task-event__sent">{this.props.event.sent}</time>
```

Comment:
Would it make sense for this component to pick a date format, rather than expecting the date to already be formatted?